pylib/requirements.txt

```diff
@@ -1,8 +1,3 @@
-# See python driver docs: futures and six have to be installed before
-# cythonizing the driver, perhaps only on old pips.
-# http://datastax.github.io/python-driver/installation.html#cython-based-extensions
-futures
```

Comment:
@bschoening sorry for bothering and changing my mind but I think we should remove only six from here, we will solve "futures" in the other ticket.